### DISCUSSION TITLE
Any way to quick unlock or not have to type in my long password every time?

### DISCUSSION THREAD

**Discussion by talormanda** [keepassxreboot/keepassxc#8742] [Q&A] [answered]
I am on 2.7.4 and I see it has a Touch ID / Windows Hello quick unlock...but when I enable it, I still have to type in my full password? Is this intended? I am looking for a way to unlock faster.

**Comment by droidmonkey** [comment, marked as answer]
Yea it is intended. If you quit keepassxc or close the database file, you will have to reenter your credentials again.

**Reply by talormanda** [reply]
Is there any way to log in faster or without typing? Such as a hardware USB key or something of the sort?

**Reply by droidmonkey** [reply]
You can set what ever password you want for your database. You can also buy a yubikey and use that with your database.

**Reply by talormanda** [reply]
How does a yubikey work if I am using it on my PC but I also want to access the DB on my phone? Can you use yubikey OR password to get in? If so, I could then still use it on my phone.

**Reply by droidmonkey** [reply]
You can using yubikey nfc and keepass2android. Keepassdx is working on that capability.

**Reply by talormanda** [reply]
I guess what I am asking is: Am I able to turn on yubikey on my PC, but still use the password to login so I can use it on my PC / phone?

1. use yubikey as the method to get in
2. still be able to use password if i want to?

or does yubikey use still require to type in the password?

**Reply by droidmonkey** [reply]
No you cannot. However you can use keeshare to export a portion of your database for mobile that only uses password for protection. Or use two separate databases and use auto open on your desktop. This is documented in the user guide.

**Reply by talormanda** [reply]
Darn, I just really hate typing in that huge password all the time. On my phone, not a big deal as I use my fingerprint as biometrics to login.

**Reply by benjuade** [reply]
> Yea it is intended. If you quit keepassxc or close the database file, you will have to reenter your credentials again.

@droidmonkey I want to better understand why and how the Cryptomator and KeepassXC Touch ID implementations differ. In fact, CM stores forever the password (unless the user removes it), while KPXC requires it at every restart of the app.

Are there any security benefits or pitfalls in the two implementations?

The CM is for sure more user-friendly, but I don’t understand if the KPXC is somehow more secure.
